src/v2/Components/NavBar/NavBarItem.tsx

```diff
@@ -0,0 +1,55 @@
+import { Clickable } from "@artsy/palette"
+import { themeGet } from "@styled-system/theme-get"
+import styled, { css } from "styled-components"
+import { RouterUnawareLink } from "v2/System/Router/RouterLink"
+
+const navBarItemMixin = css<{ active?: boolean }>`
+  position: relative;
+  display: flex;
+  height: 100%;
+  align-items: center;
+  color: ${themeGet("colors.black100")};
+  text-decoration: none;
+
+  ${props =>
+    props.active &&
+    css`
+      color: ${themeGet("colors.blue100")};
+    `}
+
+  &:hover {
+    color: ${themeGet("colors.blue100")};
+  }
+`
+
+// NOTE: Found's `Link` doesn't forward its ref, which is necessary for the UI.
```

Comment:
This is an annoying discovery. I'd PR a fix but we're already a few versions behind.